static/images/illustrations/promo-star.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="6 6 92 92" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M85.16 71.382l.898 10.75-10.78.425-3.285 10.275-10.12-3.733-6.966 8.236-7.923-7.322-9.589 4.944-4.516-9.797-10.75.898-.426-10.78-10.274-3.285 3.733-10.12-8.236-6.966 7.322-7.923-4.943-9.589 9.796-4.516-.898-10.75 10.78-.426 3.285-10.274 10.12 3.733 6.966-8.236 7.923 7.322 9.589-4.943 4.516 9.796 10.75-.898.425 10.78 10.275 3.285-3.733 10.12 8.236 6.966-7.322 7.923 4.944 9.588-9.797 4.517z" fill="#00BE28"/></svg>
```

Comment:
Seems like we're using a different color (#00BE28) from the one that's used for the Jetpack logo (#2fb41f). Is that intentional? cc @crunnells